tests/src/index.test.ts

```
@@ -280,6 +280,44 @@ describe.each(CONFIGURATIONS)(
+          await conn.getAsUpdate()
+          throw new Error('Expected error')
+        } catch (e) {
+          if (e instanceof YSweetError) {
+            expect(e.cause.code).toBe('InvalidAuthProvided')
+          } else {
+            throw e
+          }
+        }
+      })
+
+      test.only('Connecting with 5 validForSeconds should work briefly', async () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)